netlify.toml

```diff
@@ -2,8 +2,7 @@
 command = """
     set -e
     pip install -e .
-    ./scripts/build-glean-metadata
-    ./scripts/build-legacy-metadata
+    ./scripts/gd build-metadata
```

Comment:
I'm trying to find where `build-metadata` is defined so that this command invokes `__main__.py` when called... Could you help explain?

Reply:
`build-metadata` is a parameter that is passed down to the command-line logic in `__main__.py`. It's probably worth reading the changes to that file (which github has decided to hide for some reason) along with the click documentation: https://pypi.org/project/click/